test/test_orbit/test_orbit.cpp

```diff
@@ -0,0 +1,372 @@
+#include <stdio.h>
+#include <orb/Orbit.h>
+#include <cstdint>
+#include <limits>
+#include <lin.hpp>
+#include <gnc/constants.hpp>
+#include <gnc/config.hpp>
+
+
+//UTILITY MACROS
+#include <unity.h>
+#include "../custom_assertions.hpp"
+
+#define NANOSECONDS_IN_WEEK (7ULL*24ULL*60ULL*60ULL*1'000'000'000ULL)
```

Comment:
We could, and probably should, just add this to the constants generator for gnc. It's most likely going to be used in multiple places both in gnc and flight software.